ref(debug-files): Split into smaller pages (#2878)
getsentry/sentry-docs: src/docs/product/error-monitoring/reprocessing.mdx

---
title: Reprocessing
sidebar_order: 10

> description: Sentry can reprocess issues that were missing required debug information files. Learn which platforms support reprocessing and how to set it up for your projects.

Reviewer: ```suggestion
description: Learn about reprocessing, which can occur for issues missing required debug information files, including which platforms support reprocessing and how to set it up for your projects.
```
